manticore/core/smtlib/solver.py

```diff
@@ -461,7 +468,7 @@ def get_value(self, constraints, expression):
                 var = temp_cs.new_bool()
             elif isinstance(expression, BitVec):
                 var = temp_cs.new_bitvec(expression.size)
-            elif isinstance(expression, Array):
+            elif isinstance(expression, (Array, ArrayProxy)):
```

Comment:
Is the check for `ArrayProxy` necessary, as its an `Array` subclass?

Reply:
It should not be necessary! Let's remove it and see.